core/DAPS/README.md

```diff
@@ -1,16 +1,212 @@
 # Dynamic Attribute Provisioning Service (DAPS)
 
+The infrastructure component "Digital Attribute Provisioning Service" (DAPS)
+ in a given IDS ecosystem enables enrichment of identities of organisations
+ and connectors with additional attributes.
+ 
+DAPS can be understood (like the [Certification Authority](../CA/README.md), too) as building block of the construct of an
+ [IDS Identity Provider](../../glossary/README.md#identity-provider). 
+
+These attributes are embedded by the DAPS into a [Dynamic Attribute Token (DAT)]().
+ The advantages of this technique of *dynamic attribute provisioning* instead
+ of embedding them into [X.509 certificates](https://en.wikipedia.org/wiki/X.509)
+ are:
+
+- Attribute revocation does not enforce certificate revocation and re-issuance.
+ This is also true if new attributes are added. 
+- By defining scopes or attribute sets, only the needed attributes can be
+ included in the DAT. This limits information leakage, so only needed
+ attributes are communicated.
+- Baseline certificates can be issued to enable connector deployment
+ in uncomplicated manner.
+- More complex scenarios can be created as soon as attributes are
+ assigned later on. 
+
 See also:
 - [Glossary "Dynamic Attribute Provisioning Service"](../../glossary/README.md#dynamic-attribute-provisioning-service)
 - Shortcut: `DAPS`
 
+---
+
+## Unique Identifier
+
+Each (TODO: COMPONENT/entry) connector in the IDS needs a valid, outlasting and unique
```

Comment:
There is a TODO in the text

Reply:
All those "TODOs" are TODOs indeed. At the time I started those entries (keep in mind: DRAFT!!!) I tried to learn and show how it possible could look like - nothing more. The DAPS/DAT was the first one I'd started with. So those TODOs were very natural, to say so from todays perspective ;-) In the future we we will fight those TODOs in IDS-G-pre!!!